variants/NUCLEO_F756ZG/PeripheralPins.c

```diff
@@ -199,17 +199,22 @@ WEAK const PinMap PinMap_PWM[] = {
 #ifdef HAL_UART_MODULE_ENABLED
 WEAK const PinMap PinMap_UART_TX[] = {
   {PA_0,  UART4,   STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF8_UART4)},
-  {PA_2,  USART2,  STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF7_USART2)},
-  {PA_9,  USART1,  STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF7_USART1)},
+  //{PA_2,  USART2,  STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF7_USART2)},
+  //{PA_9,  USART1,  STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF7_USART1)},
+  {PA_15, UART7,   STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF12_UART7)}, // D20
```

Comment:
It looks like you mixed with some F767:
On F756 PA_15 doesn't have UART7 alternate function.
I suggest that you restart from original generated PeripheralPins.c, and then comment the useless pin (keep 1 line per pin)
Note: It causes compilation error highlighted by Continuous Integration

Reply:
This is the original generated PeripheralPins.c, I only uncommented stuff.

Reply:
No, this is not the original file, this is the original one:
https://github.com/stm32duino/Arduino_Tools/blob/master/src/genpinmap/Arduino/STM32F7/STM32F756ZGTx/PeripheralPins.c
and as you can see PA_15 has not UART7.